docs/react-oidc-context.api.md

```diff
@@ -53,18 +53,11 @@ export interface AuthContextProps extends AuthState {
 // @public
 export const AuthProvider: (props: AuthProviderProps) => JSX.Element;
 
+// Warning: (ae-forgotten-export) The symbol "AuthProviderNoUserManagerProps" needs to be exported by the entry point index.d.ts
```

Comment:
looks like AuthProviderNoUserManagerProps + AuthProviderUserManagerProps needs to be exported